earthpy/tests/test_spatial.py

```diff
@@ -18,6 +18,19 @@ def b1_b2_arrs():
     return b1, b2
 
 
+@pytest.fixture
```

Comment:
I noticed that we have a lot of tests in the spatial module. I suggest that we begin pulling out tests into individual files. Can you please remove these tests and fixtures and create a new `test_hillshade.py` script for them instead? I think in the long run this will be easier to manage.

Reply:
Done.